exercises/crypto-square/.meta/solutions/crypto_square.rb

```diff
@@ -20,6 +25,7 @@ def size
   end
 
   def ciphertext
+    return '' if normalize_plaintext == ''
```

Comment:
The canonical data also had a test which asserts that an empty array passed into ```Crypto.ciphertext``` will return an empty string. However, the current ```Crypto``` class still just blows up when it tries to call ```each_slice``` with a size of 0.